ssm-util.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
I'd consider 'auto-formatting' this in IntelliJ or something along those lines. At least, the indentation is inconsistent in `wait_for_command` which might be worth a tweak.

Minor point though and no worries.